File: Python3/프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    odd_sum, even_sum = 0, 0
    for i in range(0, len(num_list)):
        if i % 2 == 0:
            odd_sum += num_list[i]
        else:
            even_sum += num_list[i]
            
    return max(odd_sum, even_sum)

File: Swift/프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.swift
import Foundation

func solution(_ num_list:[Int]) -> Int {
    var odd_num = 0
    var even_num = 0
    
    for index in 0..<num_list.count {
        if index % 2 == 0 {
            odd_num += num_list[index]
        } else {
            even_num += num_list[index]
        }
    }
    return max(odd_num, even_num)
}